File: c#/Creational/Builder/Dto/Address.cs
namespace Dto.Address
{
    public class AddressDto
    {
        private readonly string city;
        private readonly string country;
        private readonly string cp;
        private readonly string addr;

        public AddressDto(string city, string country, string cp, string addr)
        {
            this.city = city;
            this.country = country;
            this.cp = cp;
            this.addr = addr;
        }

        public override string ToString()
        {
            return $"{addr} - {city} - {cp} - {country}";
        }
        
    }
}

File: typescript/src/creational/builder/dto/Address.ts
export class Address {
    private city: string
    private country: string
    private cp: string
    private addr: string

    constructor(city: string, country: string, cp: string, addr: string) {
        this.city = city
        this.country = country
        this.cp = cp
        this.addr = addr
    }

    toString(): string {
        return `Address: ${this.addr} - ${this.city} - ${this.cp} - ${this.country}`
    }
}